pkg/pillar/cmd/zedagent/hardwareinfo.go

```diff
@@ -98,17 +98,17 @@ func PublishHardwareInfoToZedCloud(ctx *zedagentContext) {
 		log.Fatal("PublishHardwareInfoToZedCloud proto marshaling error: ", err)
 	}
 
-	statusURL := zedcloud.URLPathString(serverNameAndPort, zedcloudCtx.V2API, devUUID, "info")
-
 	buf := bytes.NewBuffer(data)
 	if buf == nil {
 		log.Fatal("PublishHardwareInfoToZedCloud malloc error")
 	}
 	size := int64(proto.Size(ReportHwInfo))
 
-	zedcloud.SetDeferred(zedcloudCtx, hwInfoKey, buf, size,
-		statusURL, bailOnHTTPErr, false, info.ZInfoTypes_ZiHardware)
-	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, true)
+	const ignoreErr = LOCDest
+	ditems := createDeferredItemsByDest(ctx, dest, ignoreErr, hwInfoKey, buf,
+		size, bailOnHTTPErr, false, info.ZInfoTypes_ZiHardware)
+	zedcloud.AddDeferredList(zedcloudCtx, ditems)
+	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, false)
```

Comment:
Why are you changing the last argument to HandleDeferred from true to false here?